IsogeomGenerator/generate_isogeom.py

```diff
@@ -176,7 +170,7 @@ def set_moab_only_options(parser):
                         action='store',
                         nargs=1,
                         required=False,
-                        default=None,
+                        default=[None],
```

Comment:
any reason why you are using `[None]` and not just `[]`

Reply:
If it's just `[]`, wouldn't `args.savepath[0]` on new line 516 fail? Maybe that's why she changed so, although I don't know how it worked before, seeing old line 504.

Reply:
Yes @ypark234 is correct. And if the user supplies a value here, it is stored in a list, so I have to access it with `[0]` if that's the case. To avoid doing a bunch of if-statements everywhere to check if an argument was supplied or not, I can call `args.savepath[0]` in line 516 regardless of the situation. And then the code will handle if the passed value is an acceptable input or if it is `None`.